src/main/java/org/opensearch/flowframework/transport/CreateWorkflowTransportAction.java

```diff
@@ -231,7 +231,7 @@ protected void doExecute(Task task, WorkflowRequest request, ActionListener<Work
                             logger.info("updated workflow {} state to {}", request.getWorkflowId(), State.NOT_STARTED.name());
                             listener.onResponse(new WorkflowResponse(request.getWorkflowId()));
                         }, exception -> {
-                            logger.error("Failed to update workflow state : {}", exception.getMessage());
+                            logger.error("Failed to update workflow in template index: ", exception);
```

Comment:
This prints the whole exception rather than the message. Is that what's desired?

Reply:
here I wanted to print the whole exception to get the stack trace, few times I have had cases where only message is printed and its not helpful enough, core has also moved to doing this in a good amount of places